conf/local.conf.sample

```diff
@@ -93,8 +93,8 @@ USER_FEATURES += "~3g"
 # Disable near field communication support by default
 USER_FEATURES += "~nfc"
 
-# When the meta-virtualization layer is enabled, enable the feature by default
-USER_FEATURES += "${@bb.utils.contains('BBFILE_COLLECTIONS', 'virtualization-layer', 'virtualization', '', d)}"
+# Comment to disable and Uncomment to enable virtualization feature
```

Comment:
It's already uncommented, I think this is unnecessarily verbose, and also doesn't align with the wording of other entries in this file.

Reply:
Updated as per @Noor-Ahsan inputs.